frontends/web/src/hooks/qrcodescanner.ts

```diff
@@ -81,12 +85,20 @@ export const useQRCodeScanner = ({
         })
         .catch(error => {
           if (error) {
+            // Preventing error alerting when user on mobile
+            // navigates from WC connect screen to another page
+            // (this error is 'classified' as NotFoundException).
+            if (error.toString().includes('NotFoundException') && isMobile) {
+              return;
+            }
+            // Otherwise, alert as normal.
             alertUser(error.message || error);
           }
```

Comment:
nit: would it be a bit cleaner to have a onError callback instead of passing isMobile ?